Source/GTMSessionFetcher.h

```diff
@@ -384,45 +383,6 @@
 extern "C" {
 #endif
 
-#if (TARGET_OS_TV || TARGET_OS_WATCH ||                          \
-     (!TARGET_OS_IPHONE && defined(MAC_OS_X_VERSION_10_11) &&    \
-      MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_11) || \
-     (TARGET_OS_IPHONE && defined(__IPHONE_9_0) &&               \
-      __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_9_0))
-#ifndef GTM_USE_SESSION_FETCHER
-#define GTM_USE_SESSION_FETCHER 1
-#endif
```

Comment:
Given the min versions, I think this can reduce to just a #ifndef...#define...#endif

Reply:
Removed the GTM_USE_SESSION_FETCHER altogether, since I don't believe there's a point to support it being defined to 0 at this point.